config/webpack.config.shared.js

```diff
@@ -1,6 +1,11 @@
 const paths = require('./paths')
 
 module.exports = {
+  tsLoader: {
```

Comment:
This was mostly all that was needed for dev server and TypeScript builds to work. However, Jest needs a little more so wouldn't recommend writing `.ts` files yet. I however will leave the setup in here as it works other than that.

Reply:
thanks! im soooo ready to start moving to typescript